ScribusGeneratorBackend.py

```diff
@@ -214,8 +214,9 @@ def parse_data(self):
         if extension == '.csv':
             # .. from CSV file
             data = self.load_csv(data_file)
+            logging.debug(data)
 
-            if len(data) <= 1:
```

Comment:
why this change ? if data.len is 1 then there's only titles, not values to substitute > drop.